core-services/image-mirroring/integr8ly/mapping_integr8ly_operator

```diff
@@ -1,2 +1,3 @@
 registry.svc.ci.openshift.org/integr8ly/integreatly-operator:integreatly-operator quay.io/integreatly/integreatly-operator:master
-registry.svc.ci.openshift.org/integr8ly/integreatly-operator:integreatly-operator-test-harness quay.io/integreatly/integreatly-operator-test-harness:master
\ No newline at end of file
+registry.svc.ci.openshift.org/integr8ly/integreatly-operator:integreatly-operator-test-harness quay.io/integreatly/integreatly-operator-test-harness:master
+registry.svc.ci.openshift.org/integr8ly/integreatly-operator:integreatly-operator-test-harness-osde2e quay.io/integreatly/managed-api-test-harness:osde2e
```

Comment:
Why the change of name 'integreatly-operator-test-harness-osde2e' -> 'managed-api-test-harness'?

Reply:
good point, thanks, the name change isn't really required.